libfreerdp/core/rdp.c

```diff
@@ -34,7 +34,6 @@
 
 #define TAG FREERDP_TAG("core.rdp")
 
-#ifdef WITH_DEBUG_RDP
```

Comment:
I personally liked to have a way to disable this even if the WLOG_LEVEL is set to debug. 
Maybe we can extra DEBUG option for that.